StudioCore/AssetLocator.cs

```diff
@@ -780,7 +780,7 @@ public string GetParamNamesDir()
         public PARAMDEF GetParamdefForParam(string paramType)
         {
             PARAMDEF pd = PARAMDEF.XmlDeserialize($@"{GetParamdefDir()}\{paramType}.xml");
-            ParamEditor.ParamMetaData meta = ParamEditor.ParamMetaData.XmlDeserialize($@"{GetParammetaDir()}\{paramType}.xml", pd);
+            //ParamEditor.ParamMetaData meta = ParamEditor.ParamMetaData.XmlDeserialize($@"{GetParammetaDir()}\{paramType}.xml", pd);
```

Comment:
I'd prefer not leaving commented lines of code

Reply:
Hmm, I'll remove it. Not too sure I agree things of this nature (a commented out line serving as a reminder of incomplete functionality). However in this specific case, this function should be abandoned anyway (I will do this in the future) so it doesn't matter too much.

Reply:
In that case it would be good to comment what the purpose was, why it was commented out, or add a comment saying what needs to be done for the cleanup

Reply:
Fair. I'll do that if this sort of thing comes up again in the future.